src/bot.ts

```diff
@@ -33,6 +33,8 @@ export const client = new SapphireClient({
   ]
 });
 
+client.fetchPrefix = (): string => BOT_PREFIX;
```

Comment:
whats this for?

Reply:
whoops, must've slipped through when I was doing sapphire tests